index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav class="container">
+
+    <div class="time-cell-wifi-batt">
+      <div class="time">9:41</div>
+      <div class="cell"><img src="image/cell.png" alt="cell"></div>
+      <div class="wifi"><img src="image/wifi.png" alt="wifi"></div>
+      <div class="batt"><img src="image/batt.png" alt="batt"></div>
+    </div>
+    <div class="menu"><img src="image/menu-btn.png"></div>
+    <div class="logo"><img src="image/my-logo.png" alt="My Logo"></div>
+
+    <div class="container-headline">
+      <div class="primary-text">
+        Hey There. <br>
+        I'm Joshua.
+      </div>
+      <div class="supporting-text">I'm a Software Developer</div>
+      <div class="description">
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hesitate to contact me.</p>
+      </div>
+      <div class="social">
+        <div class="git"><img src="image/git.png" alt="git"></div>
+        <div class="linkedin"><img src="image/linkedin.png" alt="linkedin"></div>
+        <div class="vector"><img src="image/vector.png" alt="vector"></div>
+        <div class="twitter"><img src="image/twitter.png" alt="twitter"></div>
+        <div class="medium"><img src="image/medium.png" alt="medium"></div>
+      </div>
```

Comment:
- [ ] The social icons section is to be made using the `<ul>, <li> and <a>` tags as they should be contained as links which points to the authors social media pages. Using these semantic tags is good for seo optimization. kindly please imoplement this before making your next pull request:+1: 
- [ ] make sure to use flexbox to align the contents.